docs/guides/rookieguide.md

```diff
@@ -1,7 +1,126 @@
+
+A: [Rozwiązanie problemu znajdziesz tutaj.](../troubleshooting/acre.md)
+
+Q: Co to ACRE2 i TS?.
+
+A: TS - TeamSpeak, taki komunikator. ACRE to plugin do TSa i mod do Army3 w jednym, który ZAWSZE ZNAJDUJE SIĘ W MODLIŚCIE, więc nie musicie się martwić, że tego nie pobraliście. Macie modlistę na misję - macie ACRE. Potrzebujecie jedynie TeamSpeaka, który jest oprogramowaniem darmowym. Zalecam pierwsze odpalenie Army z nasza modlistą przeprowadzić z wyłączonym TSem. ACRE przekopiuje sobie wtedy swoje pliki do katalogów TSa, w innym wypadku nie będzie miało do nich dostępu, bo będą używane przez włączonego TSa. 
+
+TeamSpeaka możecie nawet odpalić w trakcie gry, ale robimy to z reguły przed lub po wejściu do lobby na serwerze. Kiedy już Zeus (Game Master) odpali misję, to automatycznie przerzuci was do odpowiedniego kanału na TSie.
+
+ACRE symuluje komunikację jak w prawdziwości - jeżeli ktoś jest daleko, to usłyszycie go tylko jak krzyknie przy odpowiednio podregulowanej głośności. Są też radia, czyli podstawa łańcucha dowidzenia. Tutaj odsyłam do [poradnika poświęconego ACRE2](../guides/acre), gdzie znajdziecie też informacje na temat obsługi radia. Na serwerze zawsze ktoś wam z tym pomoże jeżeli poprosicie, ale przydaje się wcześniejsza świadomość istnienia takiej mechaniki.
+
+Jeżeli macie problemy z ACRE i TS to przeczytajcie ponownie poświęconą temu część poradnika.
+
+Q: Na stronie z modlistą świeci się na żółto jedna z paczek. Co to znaczy?
+
+A: Że ta żółta paczka jest obecnie załadowana na serwerze. Nie uda ci się dołączyć do serwera odpalając grę z inną paczką modów, niż jest obecnie załadowana. Gra będzie sypać błędami. Paczka musi być aktualna, więc najlepiej parę godzin przed misją pobrać paczkę jeszcze raz, żeby mieć pewność, że wszystko się zaktualizowało. Raz na ruski rok mission maker zmieni zawartość paczki na chwilę przed misją (np. godzinę), wtedy jest dawany cynk na discordzie. Ale ta sytuacja dzieje się dość rzadko.
+
+Q: Bardzo dużo teorii, a ja nie potrafię szybko czytać. Czy mogę jakoś w praktyce wypróbowac zaprezentowaną tu wiedzę?
+
+A: Tak, serwer jest z reguły odpalony 24/7, po uzyskaniu rangi rekruta możesz normalnie popróbować wejść na serwer, jeżeli jakiś mission maker nie testuje misji, to odpalony jest default modlist (ale jak zawsze odsyłam do sprawdzenia “co się świeci” na stronie z  modlistami) na mapie Altis - można sobie pojeździć i polatać, poleczyć boty. Są spawnery tu i tam.
+Zachęcam również do sprawdzenia działania ACRE/TeamSpeaka, na serwer TS też możecie wchodzić bez strachu, czujcie się jak u siebie. ACRE najlepiej przetestować z drugą osobą, poproście rekruterów lub innych rekrutów, bo może też szukają partnera/ki do testów.
+
+Q: Ile przeciętnie trwa misja?
+
+A: Od 1,5h do 2,5h. Jeżeli coś się wysypie - może trwać dłużej. Misje, które są JointOps z innymi grupami zazwyczaj trwają dłużej, czasem nawet do 4 godzin.
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Nit: Is this break intended?

Reply:
I think I wanted to put something there but eventually changed my mind. There is no need for it now so it can be deleted

Reply:
Resolved